libcontainer/configs/validate/validator.go

```diff
@@ -353,3 +355,31 @@ func isHostNetNS(path string) (bool, error) {
 
 	return (st1.Dev == st2.Dev) && (st1.Ino == st2.Ino), nil
 }
+
+// scheduler is to validate scheduler configs according to https://man7.org/linux/man-pages/man2/sched_setattr.2.html
+func scheduler(config *configs.Config) error {
+	if config.Scheduler == nil {
```

Comment:
I think we can simplify the code by adding something like
```go
s := config.Scheduler
```

and then using `s` everywhere, for example:
```diff
-niceValue := config.Scheduler.Nice
-if niceValue < -20 || niceValue > 19 {
-	return fmt.Errorf("invalid scheduler.nice: %d", niceValue)
+ if s.Nice < -20 || s.Nice > 19 {
+	return fmt.Errorf("invalid scheduler.nice: %d", s.Nice)
```